### DISCUSSION TITLE
Resource page - disable create button

### DISCUSSION THREAD

**Discussion by Grazulex** [filamentphp/filament#3774] [Help] [answered]
Hello, 
I have read the documentation and remove the "create" acrion from the getPages function : 

```
    public static function getPages(): array
    {
        return [
            'index' => Pages\ListProductions::route('/'),
            'edit' => Pages\EditProduction::route('/{record}/edit'),
        ];
    }   
``` 

and remove the file in the pages folder but I have still the button and when I click on it I have the form (?!).
How can I remove the "create" button from a resource page ?

thank

**Comment by RX-77IT** [comment, marked as answer]
Very strange...

Does adding this function in the Resource works?

`public static function canCreate(): bool
    {
        return false;
    }`

**Reply by Grazulex** [reply]
yes, many thanks

**Reply by danharrin** [reply]
You can just remove the `CreateAction::make()` from the List page's `getActions()`... no need to override methods.

**Reply by Grazulex** [reply]
I agree; it's working too
